k8s/k8s.go

```diff
@@ -4487,6 +4491,29 @@ func (k *k8sOps) UpdateBackupLocation(backupLocation *v1alpha1.BackupLocation) (
 	return k.storkClient.Stork().BackupLocations(backupLocation.Namespace).Update(backupLocation)
 }
 
+func (k *k8sOps) ValidateBackupLocation(name, namespace string, timeout, retryInterval time.Duration) error {
+	t := func() (interface{}, bool, error) {
+		if err := k.initK8sClient(); err != nil {
+			return "", true, err
+		}
+
+		resp, err := k.GetBackupLocation(name, namespace)
+		if err != nil {
+			return "", true, err
+		}
```

Comment:
instead of just checking we can do a get, we should also validate the response as the APIs intention is validate.

Reply:
There is nothing to validate, there is no status for the backupLocation

Reply:
We can atleast check if `resp.Location` is populated.

Reply:
This will just give false positives. Having anything set doesn't mean its valid.

Reply:
Fixed.